sympy/physics/continuum_mechanics/beam.py

```diff
@@ -3,7 +3,6 @@
 singularity functions in mechanics.
 """
 
-from __future__ import print_function, division
 
```

Comment:
Here and many other places

Reply:
The line from which imports are removed are also removed automatically.

Reply:
Next line!

Reply:
There are 214 files that are changed and is done by tool , so I dont think it's necessary to remove each line by hand.

Reply:
The fact that this is done automatically does not mean that it isn't necessary to check manually as well afterwards. @sidhu1012 have you been through the diff for this PR checking whether the line spacing is consistent with the linespacing in these files before applying the script?

Reply:
Yes

Reply:
The examples highlighted by @smitgajjar seem to be cases where the new linespacing is not consistent with how it was before.